maintenance/wikia/setWikiFactoryVariable.php

```diff
@@ -125,6 +135,34 @@ public function execute() {
 				echo "Previous value: " . $prevValue . PHP_EOL;
 				echo "New value: " . $newValue . PHP_EOL;
 				$status = $this->setVariable( $id, $newValue );
+			} else if ( $merge ) {
+				$varData = (array) WikiFactory::getVarByName( $this->varName, $id, true );
+				if (!empty ($split)) {
+					// value is string so we must convert to array
+					$prevValue = unserialize($varData['cv_value']);
+					$prevValue = explode($split, $prevValue);
+				}
+
+				if (is_array($prevValue)) {
+					echo "Adding new value $varValue to array" . PHP_EOL;
+					$newValue = array_filter(array_unique(array_merge($prevValue, [$varValue])));
+				} else {
+					echo "Value is not an array" . PHP_EOL;
```

Comment:
What do you think about a little bit different copy: `Value is not an array. Have you forgotten about --split parameter?` :)